src/cms/contentful.ts

```diff
@@ -98,11 +146,25 @@ export class HarmonizerContentfulClient extends AbstractAgnosticCMSHarmonizerCli
     return this.getClientInstance().getEntry<ContentfulEntrySkeleton<T>>(entryId, query);
   }
 
+  private async getEntriesHandler<T = Record<string, unknown>>(
+    query?: EntriesQueries<ContentfulEntrySkeleton<T>, undefined>,
+  ): Promise<ContentfulEntry<T>[]> {
+    const entries = await this.getClientInstance().getEntries<ContentfulEntrySkeleton<T>>(query);
```

Comment:
As [the API returns the items paginated](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/entries/entries-collection/get-all-entries-of-a-space/console/js), shouldn't we also return them paginated?

If we don't want to return them paginated, then we need to internally obtain all the items recursively and return all of them.

Reply:
This PR does not intended to implement that feature I'm afraid. For now, I'd say it is what it is and can add the pagination to the roadmap. Would there be any minimal change that easily enables collecting all records? 🤔

Reply:
We could set `[limit](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters/limit)` to 1000 (it's the maximum), so at least for most of the cases we make sure we're returning all the elements. In fact 100 is a lot, but just in case